pypy/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM pypy:3
+LABEL maintainer="Saksham Phul <phuls@chop.edu>"
+LABEL description="PyPy docker image with numpy, pandas, and scipy with their dependencies"
+
+WORKDIR /home
+
+ENV NUMPY_VERSION 1.23.0
+ENV SCIPY_VERSION 1.8.1
+ENV PANDAS_VERSION 1.4.3
+ENV PYREADR_VERSION 0.4.6
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/OpenBLAS/lib" 
+
+RUN     apt-get update \ 
+    && apt-get install gfortran gfortran-10 libgfortran-10-dev libgfortran5 \ 
```

Comment:
Not really critical, more of a style thing - just `apt` works the same as `apt-get`